common/source/docs/common-install-sdcard.rst

```diff
@@ -92,7 +91,7 @@ After the transfer is complete, the directory listing should look something like
 Triggering the Flash Update
 ===========================
 
-Power cycle the board to enter the bootloader which will automaticallt check for the firmware update file and begin flashing it.
+Power cycle the board to enter the bootloader which will automatically check for the firmware update file and begin flashing it.  Alternatively to avoid the powercycle the autopilot may be rebooted via a `PREFLIGHT_REBOOT_SHUTDOWN <https://mavlink.io/en/messages/common.html#MAV_CMD_PREFLIGHT_REBOOT_SHUTDOWN>`__ command with the 'Param1' field set to 3 (e.g. "Reboot autopilot and keep it in the bootloader until upgraded")
```

Comment:
```suggestion
Power cycle the board to enter the bootloader which will automatically check for the firmware update file and begin flashing it.  Alternatively to avoid the powercycle the autopilot may be rebooted via a `PREFLIGHT_REBOOT_SHUTDOWN <https://mavlink.io/en/messages/common.html#MAV_CMD_PREFLIGHT_REBOOT_SHUTDOWN>`__ command with the 'Param1' field set to 3 (ie. "Reboot autopilot and keep it in the bootloader until upgraded")
```